glance/templates/api/deployment.yaml.j2

```diff
@@ -7,9 +7,6 @@ spec:
       labels:
         app: glance-api
     spec:
-      # TODO(DTadrzak): it must be removed in future
```

Comment:
Hey,

/etc/ceph/ceph.conf directory is not writeable, by root. We need to fix that at the image level @DTadrzak